examples/src/RecordMovieComponent.tsx

```diff
@@ -9,8 +9,17 @@ const RecordMovieComponent = ({
     isRecording,
     setIsRecording,
 }: RecordMovieComponentProps) => {
+    // recording time measured in seconds
     let [recordingTimeElapsed, setRecordingTimeElapsed] = useState<number>(0);
     let [outputStatus, setOutputStatus] = useState<string>("");
+    let [browserSupported, setBrowserSupported] = useState<boolean>(false);
+
+    useEffect(() => {
+        if ("VideoEncoder" in window) {
+            setBrowserSupported(true);
+        }
+    }
+    , []);
```

Comment:
Does this need to be a state variable? Can it just be a one-line `const browserSupported = "VideoEncoder" in window`?